STYLEGUIDE.md

```diff
@@ -190,7 +190,22 @@ Keeps functions pure.
 
 ---
 
-### Use “be” verbs for boolean types. (e.g. `should`/`is`/`can`/`does`/`has`)
+### Prefix boolean variables with “to be” verbs.
+
+(e.g. `shouldX`, `isY`, `canZ`, `doesX`, `hasY`, `willZ`)
+
+---
+
+### Follow BEM-ish patterns when hard-coding a `data-testid` or `data-lgid`
+
+1. Prefix with “lg”
+2. Words are separated by underscore `_`
+3. Blocks are separated by dash `-`
+4. Name hierarchy should somewhat match the directory structure
+
+e.g. `data-testid="lg-date_picker-calendar_cell"`
+
+Additionally, a hard-coded `data` attribute should _only_ be placed on a root-level, native HTML element. Components should have their own internal id.
```

Comment:
we should have a `prefer/not` section

Reply:
What do you mean by that?

Reply:
As in "not `lg-date-picker-calendar-cell` (with dashes)" ?

Reply:
https://github.com/mongodb/leafygreen-ui/pull/2221/files/4638ff2c841a9bd9ff9716b030ecaaae90515448#diff-18b19378fcb4038a2ce14f3ef8dd8ffa0bacd8f4e795cef51d08f3178d7e946cL20-L46

Reply:
take a look at our "Contributing to the style guide" guidelines

Reply:
I'm a bit confused. How would you write it?

Reply:
Prefer 
this_pattern-pattern

Avoid
this-pattern-pattern